### PR TITLE
Support for relative URL

### PR DESCRIPTION
**Pull request by gbugniot**
Add support for relative URL when parsing OPDS feeds. Fix readium/r2-testapp-swift#65.

### CODE REVIEW

File: readium-opds.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -11,13 +11,14 @@
 		111E54CE201BC50E00A7FE9A /* OPDSParser.swift in Sources */ = {isa = PBXBuildFile; fileRef = F34B7E681FA35BB100534FD3 /* OPDSParser.swift */; };
 		111E54CF201BC52400A7FE9A /* AEXML.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F34B7E6C1FA35CEA00534FD3 /* AEXML.framework */; };
 		111E54D0201BC52600A7FE9A /* R2Shared.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = F34B7E6B1FA35CEA00534FD3 /* R2Shared.framework */; };
+		111E54D5201FB0AD00A7FE9A /* PromiseKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 111E54D4201FB0AC00A7FE9A /* PromiseKit.framework */; };
+		111E54D6201FB0B600A7FE9A /* PromiseKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 111E54D4201FB0AC00A7FE9A /* PromiseKit.framework */; };
+		111E54DA201FC91600A7FE9A /* feedbooks_catalog.atom in Resources */ = {isa = PBXBuildFile; fileRef = 111E54D9201FC91600A7FE9A /* feedbooks_catalog.atom */; };
```
Comment (aferditamuriqi) [inline readium-opds.xcodeproj/project.pbxproj]: the atom file here will show as missing file in Xcode

File: readium-opds/Tool.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
+//
+//  Tool.swift
```
Comment (aferditamuriqi) [inline readium-opds/Tool.swift]: can we find a better name for this class? Tool doesn't really tell much about the functionality. Maybe something like URLHelper could work better.

Comment (gbugniot) [reply readium-opds/Tool.swift]: Yes, agreed.

File: readium-opds.xcodeproj/project.pbxproj (hunk 1 above)
Comment (gbugniot) [reply readium-opds.xcodeproj/project.pbxproj]: After cloning this repo, the atom file doesn't show as missing, but the parent folder named "Samples" is. I've updated the folder path in Xcode. Should be fine now.

**Pull request merged**